### DISCUSSION TITLE
超级密集的文本行怎么检测，就是文本行之间近乎没有行距的这种类型

### DISCUSSION THREAD

**Discussion by MvZpt** [PaddlePaddle/PaddleOCR#13426] [Q&A] [answered]
这种图片中近乎没有行距的文本怎么检测
![45](https://github.com/user-attachments/assets/25474c21-78ca-44d4-bdd3-0e68b4b0df80)
这是我提升亮度对比度之后的检测效果

**Comment by MvZpt** [comment]
![56_a](https://github.com/user-attachments/assets/17c2d9af-b03f-4b45-a4c6-04a34f7865cb)
这是我提升亮度对比度之后的检测效果

**Comment by GreatV** [comment, marked as answer]
建议在自己的场景下，重新训练一下检测模型。
